lib/kramdown_rpf/rpf.rb

```diff
@@ -1,8 +1,15 @@
+require 'kramdown'
+
 module RPF
   module Plugin
     module Kramdown
       YAML_FRONT_MATTER_REGEXP = /\n\s*---\s*\n(.*?)---(.*)/m
-      RADIO_REGEXP = /\(\s?\)\s*(?<text>.*)/
+      RADIO_REGEXP = /\((?<check>[\sx*]{0,1})\)\s*(?<text>.*)/m
+      CHOICE_BLOCK_REGEXP = %r{^(?=#{::Kramdown::Parser::Kramdown::OPT_SPACE}- \([\sx*]?\)\s*.*)}m
+      FEEDBACK_REGEXP_PARTIAL = "#{::Kramdown::Parser::Kramdown::OPT_SPACE}---[ \t]*feedback[ \t]*---(.*?)---[ \t]*\/feedback[ \t]*---"
+      CHOICE_FEEDBACK_REGEXP = %r{#{FEEDBACK_REGEXP_PARTIAL}}m
+      SINGLE_FEEDBACK_REGEXP = %r{\A#{FEEDBACK_REGEXP_PARTIAL}}m
+      QUESTION_REGEXP = %r{(.*?)^#{::Kramdown::Parser::Kramdown::OPT_SPACE}---[ \t]*choices[ \t]*---(.*?)---[ \t]*\/choices[ \t]*---}m
```

Comment:
These should all be frozen, as they're constants.

```suggestion
      RADIO_REGEXP = /\((?<check>[\sx*]{0,1})\)\s*(?<text>.*)/m.freeze
      CHOICE_BLOCK_REGEXP = %r{^(?=#{::Kramdown::Parser::Kramdown::OPT_SPACE}- \([\sx*]?\)\s*.*)}m.freeze
      FEEDBACK_REGEXP_PARTIAL = "#{::Kramdown::Parser::Kramdown::OPT_SPACE}---[ \t]*feedback[ \t]*---(.*?)---[ \t]*\/feedback[ \t]*---".freeze
      CHOICE_FEEDBACK_REGEXP = %r{#{FEEDBACK_REGEXP_PARTIAL}}m.freeze
      SINGLE_FEEDBACK_REGEXP = %r{\A#{FEEDBACK_REGEXP_PARTIAL}}m.freeze
      QUESTION_REGEXP = %r{(.*?)^#{::Kramdown::Parser::Kramdown::OPT_SPACE}---[ \t]*choices[ \t]*---(.*?)---[ \t]*\/choices[ \t]*---}m.freeze
```